app/Http/Requests/ImageRequest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Anik\Form\FormRequest;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Contracts\Validation\Validator;
+use Illuminate\Validation\ValidationException;
+
+class ImageRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    protected function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    protected function rules(): array
+    {
+        switch ($this->method()) {
+
+            case 'POST': {
+                    return [
+                        'page_name' => 'required|string'
+                    ];
+                    break;
+                }
+            default:
+                break;
+        }
+    }
+
+    /**
+     * format the validation response if there is error in validation.
+     *
+     * @return array
+     */
+    protected function errorResponse(): ?JsonResponse
+    {
+
+        return response()->json([
```

Comment:
Can be replaced with `ResponseSeriviceProvider`

Reply:
Unable to update, package do now allow it due to compatibility issues, gives following exception

Declaration of App\Http\Request\AdsRequest::errorResponse(): ?App\Providers\ResponseServiceProvider must be
compatible with Anik\Form\FormRequest::errorResponse(): ?Illuminate\Http\JsonResponse (500 Internal Server
Error)

We are using package, we are using following package

https://github.com/ssi-anik/form-request